public/src/components/calls/pre_call_btn_grp.jsx

```diff
@@ -46,38 +46,57 @@ export default class PreCallButtonGroup extends Component {
             releaseCall,
             status,
             endVolunterTwilioCon,
-            clearCurrentCall } = this.props;
+            clearCurrentCall,
+            setNoCallsAvailable } = this.props;
     clearCurrentCall();
     releaseCall(user_id, campaign_id, call_id, status);
     endVolunterTwilioCon(user_id, campaign_id);
+    setNoCallsAvailable(false);
```

Comment:
Is this necessary? I'm imagining a scenario where the user is in the pre-call page and gets a notification that there are no calls, why are we toggling it back to false?